client/my-app/src/api/expenses.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import { BASE_URL } from '../constants';
+
+const ExpensesBaseUrl = BASE_URL + '/expenses'
+
+type GetAllProps = {
+  userId: number | null,
+  category: string,
+  from: string,
+  to: string,
+};
+
+export const getExpenses = async({ userId, category, from, to }: GetAllProps) => {
+  try {
+    const response = await axios.get(ExpensesBaseUrl, { params: { userId, category, from, to } })
+
+    return response.data;
+  } catch (err: any) {
+    throw new Error(err);
+  }
```

Comment:
this catch is useless
we catch and throw the same error
it could be catched to to modify error and add more description or throwing custom error, but now this fragment is useless